src/components/ActivityFeed/ActivityFeedTreasury.tsx

```
@@ -396,14 +424,14 @@ const ActivityFeedSidebar = () => {
 					<span className={' flex text-xs font-normal leading-5 text-lightBlue dark:text-blue-dark-medium'}>{chainProperties[network]?.tokenSymbol} Price</span>
 					<div className='flex items-center gap-x-1 text-lg font-semibold'>
 						<div>
-							{currentTokenPrice?.value === 'N/A' ? (
-								<span className=' text-bodyBlue dark:text-blue-dark-high'>N/A</span>
+							{!tokenLoading && tokenPrice ? (
+								<span className='ml-[2px] mt-1 text-bodyBlue dark:text-blue-dark-high'>${tokenPrice}</span>
 							) : currentTokenPrice?.value && !isNaN(Number(currentTokenPrice?.value)) ? (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)